File: python/examples/4.ServerAlwaysUp/run_client.py
from ezpysocket import ezpysocket as ps
import cv2
import numpy as np

if __name__ == "__main__":

    # Init
    c = ps.EzPySocket(server_mode=False)

    # Load image
    img = cv2.imread("../resources/lena.jpg", 1)

    # Sending 
    print("Sending image...")
    c.send_image(img)

    # Receiving image
    print("Receiving image...")
    img = c.receive_image()

File: cpp/examples/4.ServerAlwaysUp/run_client.cpp
#include "ezcppsocket.h"

int main(int argc, char const *argv[])
{
	EzCppSocket c = EzCppSocket("127.0.0.1", 10000, 2, 1, false, true, 1, false);

	// Sending
	std::cout << "Sending image...\n";
	cv::Mat img = cv::imread("../resources/lena.jpg");
	c.sendImage(img);

	// Receiving
	std::cout << "Receiving data...\n";
	cv::Mat recv_img = c.readImage();

	return 0;
}